tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -149,32 +149,70 @@ For safety, the "official target" version that the azure-sdk team uses is presen
 
 ## Command line arguments
 
-This is the help information for test-proxy. It uses the nuget package [`CommandLineParser`](https://www.nuget.org/packages/CommandLineParser) to parse arguments.
+This is the help information for test-proxy. It uses the nuget package [`System.CommandLine`](https://www.nuget.org/packages/System.CommandLine) to parse arguments.
 
 The test-proxy executable fulfills one of two primary purposes:
 
 1. The test-proxy server (the only option up to this point)
-2. [`asset-sync`](#asset-sync-retrieve-external-test-recordings) push/restore/reset.
+2. [`asset-sync`](#asset-sync-retrieve-external-test-recordings) verbs
+   - `push`
+   - `restore`
+   - `reset`
+   - `config`
 
 This is surfaced by only showing options for the default commands. Each individual command has its own argument set that can be detailed by invoking `test-proxy <command> --help`.
 
 ```text
-/>test-proxy --help
-Azure.Sdk.Tools.TestProxy 1.0.0-dev.20220926.1
-c Microsoft Corporation. All rights reserved.
-
-  start      (Default Verb) Start the TestProxy.
-
-  push       Push the assets, referenced by assets.json, into git.
-
-  reset      Reset the assets, referenced by assets.json, from git to their original files referenced by the tag. Will prompt
-             if there are pending changes.
-
-  restore    Restore the assets, referenced by assets.json, from git.
+/>Azure.Sdk.Tools.TestProxy --help
+Description:
+  This tool is used by the Azure SDK team in two primary ways:
+
+    - Run as a http record/playback server. ("start" / default verb)
+    - Invoke a CLI Tool to interact with recordings in an external store. ("push", "restore", "reset", "config")
+
+Usage:
+  Azure.Sdk.Tools.TestProxy [command] [options]
```

Comment:
Should this still be `test-proxy` assuming most are encouraged to use the standalone executable?

Reply:
It's not here?

Reply:
Yeah but if you have this installed standalone or via dotnet tool won't `test-proxy --help` print out `Azure.Sdk.Tools.TestProxy` as the example command with this change?